reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -114,6 +135,32 @@ public void incrementErrorsCount(SocketAddress remoteAddress, String uri) {
 		}
 	}
 
+	@Override
+	public void incrementActiveConnections(String uri, String method, int amount) {
+		MeterKey meterKey = new MeterKey(uri, null, method, null);
+		Counter activeConnections = activeConnectionsCache.get(meterKey);
+		activeConnections = activeConnections != null ? activeConnections : activeConnectionsCache.computeIfAbsent(meterKey,
+				key -> filter(activeConnectionsBuilder.tags(URI, uri, METHOD, method)
+						.register(REGISTRY)));
+
+		if (activeConnections != null) {
+			activeConnections.increment(amount);
+		}
+	}
+
+	@Override
+	public void incrementServerConnections(SocketAddress localAddress, int amount) {
+		String address = Metrics.formatSocketAddress(localAddress);
+		MeterKey meterKey = new MeterKey(null, address, null, null);
+		Counter connections = connectionsCache.get(meterKey);
+		connections = connections != null ? connections : connectionsCache.computeIfAbsent(meterKey,
+						key -> filter(connectionsBuilder.tags(URI, HTTP, LOCAL_ADDRESS, address).register(REGISTRY)));
```

Comment:
should we replace `URI` with a new `PROTOCOL` tag?

Reply:
note that if this was actually an URI, there would again be a potential for a high number of variations. not as much as the remote address, but still in some case there can be a huge number of URIs served by the server.

but in any case, this doesn't seem to be used as uri, more as a protocol tag ✅

Reply:
disregard these comments. the common API with tcp base doesn't allow passing of an URI and so the URI tag doesn't make sense here. our meters use `URI = protocol` as a convention when the uri is unknown hence this tag.